cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -0,0 +1,964 @@
+	type AdminOrigin = EnsureRoot<AccountId>;
+	type MaxRemoteLockConsumers = ConstU32<0>;
+	type RemoteLockConsumerIdentifier = ();
+}
+
+impl cumulus_pallet_xcm::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type XcmExecutor = XcmExecutor<XcmConfig>;
+}
+
+pub type ForeignCreatorsSovereignAccountOf = (
+	SiblingParachainConvertsVia<Sibling, AccountId>,
+	AccountId32Aliases<RelayNetwork, AccountId>,
+	ParentIsPreset<AccountId>,
+);
+
+/// Simple conversion of `u32` into an `AssetId` for use in benchmarking.
+pub struct XcmBenchmarkHelper;
+#[cfg(feature = "runtime-benchmarks")]
+impl pallet_assets::BenchmarkHelper<MultiLocation> for XcmBenchmarkHelper {
+	fn create_asset_id_parameter(id: u32) -> MultiLocation {
+		MultiLocation { parents: 1, interior: X1(Parachain(id)) }
+	}
+}
+
+#[cfg(feature = "runtime-benchmarks")]
+pub struct BenchmarkMultiLocationConverter<SelfParaId> {
+	_phantom: sp_std::marker::PhantomData<SelfParaId>,
+}
+
+#[cfg(feature = "runtime-benchmarks")]
+impl<SelfParaId>
+	pallet_asset_conversion::BenchmarkHelper<MultiLocation, sp_std::boxed::Box<MultiLocation>>
+	for BenchmarkMultiLocationConverter<SelfParaId>
+where
+	SelfParaId: Get<ParaId>,
+{
+	fn asset_id(asset_id: u32) -> MultiLocation {
+		MultiLocation {
+			parents: 1,
+			interior: X3(
+				Parachain(SelfParaId::get().into()),
+				PalletInstance(<Assets as PalletInfoAccess>::index() as u8),
+				GeneralIndex(asset_id.into()),
+			),
+		}
+	}
+	fn multiasset_id(asset_id: u32) -> sp_std::boxed::Box<MultiLocation> {
+		sp_std::boxed::Box::new(Self::asset_id(asset_id))
+	}
+}
+
+/// All configuration related to bridging
+pub mod bridging {
+	use super::*;
+	use assets_common::{matching, matching::*};
+	use parachains_common::xcm_config::LocationFilter;
+	use sp_std::collections::btree_set::BTreeSet;
+
+	// common/shared parameters for Wococo/Rococo
+	parameter_types! {
+		pub SiblingBridgeHubParaId: u32 = match Flavor::get() {
+			RuntimeFlavor::Rococo => bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID,
+			RuntimeFlavor::Wococo => bp_bridge_hub_wococo::BRIDGE_HUB_WOCOCO_PARACHAIN_ID,
+		};
+		pub SiblingBridgeHub: MultiLocation = MultiLocation::new(1, X1(Parachain(SiblingBridgeHubParaId::get())));
+		/// Router expects payment with this `AssetId`.
+		/// (`AssetId` has to be aligned with `BridgeTable`)
+		pub XcmBridgeHubRouterFeeAssetId: AssetId = TokenLocation::get().into();
+		/// Price per byte - can be adjusted via governance `set_storage` call.
+		pub storage XcmBridgeHubRouterByteFee: Balance = TransactionByteFee::get();
+	}
+
+	pub mod to_wococo {
+		use super::*;
+
+		parameter_types! {
+			pub SiblingBridgeHubWithBridgeHubWococoInstance: MultiLocation = MultiLocation::new(1, X2(Parachain(SiblingBridgeHubParaId::get()), PalletInstance(46)));
+
+			pub const WococoNetwork: NetworkId = NetworkId::Wococo;
+			pub AssetHubWococo: MultiLocation =  MultiLocation::new(2, X2(GlobalConsensus(WococoNetwork::get()), Parachain(1000)));
```

Comment:
I know what you mean, but here we define MultiLocation to different consensus `parents: 2` so it could be maybe strange to use this runtime's `Get<ParaId>`.

but at least I re-used const from `bp_asset_hub_*::ASSET_HUB_*_PARACHAIN_ID` instead of 1000